### ISSUE TITLE
[RNIB] 4.11 Text description transformation

### ISSUE THREAD

**Issue by RealJoshue108** [w3c/apa#91]
Filed on behalf of RNIB: [Original comments sent to APA list ](https://lists.w3.org/Archives/Public/public-apa/2020Apr/0008.html) 

Regarding User Need 10: Using simple language and avoiding large blocks of text (which can feel daunting) can also help deaf people with limited written language skills and other people with print disabilities such as dyslexia or cognitive impairments.

**Comment by RealJoshue108** [comment]
@Raisonali This is something that is covered in other accessibility guidelines. The user need/requirements come from the need to transform text into other formats. I'm wondering if the suggestion here is useful, as it may be covered in other guidance. What do you think?

**Comment by Raisonali** [comment]
I'm assuming that this document will be linked to other useful guidelines i.e., accessibility guidelines/ cross referenced in which case perhaps avoid the duplication.

**Comment by RealJoshue108** [comment]
Yes, this cross linking and referencing will be used as needed @Raisonali
